client/src/App.js

```diff
@@ -1,20 +1,47 @@
 import React, { Component } from "react";
 import { Route } from "react-router-dom";
 import SplashPage from "./pages/SplashPage/SplashPage";
+import ComingSoonPage from "./pages/ComingSoonPage/ComingSoonPage";
 import "./App.css";
-import { library } from '@fortawesome/fontawesome-svg-core'
-import { faCheckSquare, faHeart, faTable, faComment, faCaretRight, faCaretLeft, faSearch, faAngleLeft, faAngleRight, faCommentDots } from '@fortawesome/free-solid-svg-icons'
-import { fab } from '@fortawesome/free-brands-svg-icons'
-import { fas } from '@fortawesome/free-solid-svg-icons'
-import { far } from '@fortawesome/free-regular-svg-icons'
-library.add(fab, far, fas, faCheckSquare, faHeart, faTable, faComment, faCaretRight, faCaretLeft, faSearch, faAngleLeft, faAngleRight, faCommentDots  )
+import { library } from "@fortawesome/fontawesome-svg-core";
+import {
+  faCheckSquare,
+  faHeart,
+  faTable,
+  faComment,
+  faCaretRight,
+  faCaretLeft,
+  faSearch,
+  faAngleLeft,
+  faAngleRight,
+  faCommentDots
+} from "@fortawesome/free-solid-svg-icons";
+import { fab } from "@fortawesome/free-brands-svg-icons";
+import { fas } from "@fortawesome/free-solid-svg-icons";
+import { far } from "@fortawesome/free-regular-svg-icons";
+library.add(
+  fab,
+  far,
+  fas,
+  faCheckSquare,
+  faHeart,
+  faTable,
+  faComment,
+  faCaretRight,
+  faCaretLeft,
+  faSearch,
+  faAngleLeft,
+  faAngleRight,
+  faCommentDots
+);
 
 class App extends Component {
   render() {
     return (
-      <div className='App'>
+      <div className="App">
         <main>
           <Route path="/" exact component={SplashPage}></Route>
+          <Route path="/comingsoon" exact component={ComingSoonPage}></Route>
```

Comment:
Could we have the path be `coming-soon`?

Reply:
or `coming_soon`

Reply:
Lol, I'll fix it, thanks 👍